brayns/engine/Camera.h

```diff
@@ -39,6 +40,12 @@ struct LookAt
     Vector3f up{0.f, 1.f, 0.f};
 };
 
+BRAYNS_JSON_ADAPTER_BEGIN(LookAt)
+BRAYNS_JSON_ADAPTER_ENTRY(position, "Position of the camera")
+BRAYNS_JSON_ADAPTER_ENTRY(target, "Target position at which the camera is looking")
+BRAYNS_JSON_ADAPTER_ENTRY(up, "Up vector to compute the camera orthonormal basis")
+BRAYNS_JSON_ADAPTER_END()
```

Comment:
Maybe we can put the brayns adapters / messages which are exposed to plugins in a separated folder at some point.
So the engine objects don't depend directly on JSON.

Reply:
Yes, that can be moved outside at a point.